apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/SurveyAnalysisCTA.test.tsx

```
@@ -1,450 +1,432 @@
+      </div>
+    ),
+  })
+);
+
+vi.mock(
+  "@/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/share-survey-modal",
+  () => ({
+    ShareSurveyModal: ({ survey, open, setOpen, modalView, user }: any) => (
+      <div data-testid="share-survey-modal" data-open={open} data-modal-view={modalView}>
+        Share Survey Modal for {survey.id} - User: {user.id}
+        <button onClick={() => setOpen(false)}>Close Modal</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)